### PR TITLE
Added tagline and buttons to the above the fold content; Modified the Contact Us section

### PR DESCRIPTION
**Pull request by cuchoi**
1. Added tagline and buttons to the above the fold content : 
image

2. Added a link to a Google Form in the Contact Us section:
image

3. Also updated the instruction for running Docker and added a make command.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,5 +2,11 @@ serve:
 	docker run -it -p 4000:4000 --rm --mount type=bind,source="$(PWD)",target=/srv/jekyll jekyll/jekyll:latest /bin/bash
 	# docker run --mount type=bind,source="$(pwd)",target=/srv/jekyll/ --rm -it -p 4000:4000 -env JEKYLL_ENV=production jekyll/jekyll:latest /bin/bash
 
+serve-locally:
```
Comment (canyon289) [inline Makefile:5]: Thanks, I got real lazy with this one so i appreciate you making this much nicer

**Review by canyon289 — APPROVED** [review]

**Comment by OriolAbril** [comment]
Thanks! I think this will make the website easier to navigate

Given that we have 3 different types of content and one of them is called talk, I would use a different text for the "give a talk" button, for example "submit a proposal" or "speak at pymcon". Otherwise I would expect to go to a page specific for talks a la jupytercon

**Comment by canyon289** [comment]
> Thanks! I think this will make the website easier to navigate
> 
> Given that we have 3 different types of content and one of them is called talk, I would use a different text for the "give a talk" button, for example "submit a proposal" or "speak at pymcon". Otherwise I would expect to go to a page specific for talks a la jupytercon

@cuchoi going to merge so we get this out there, but would you mind making this change in another PR? And if you can't drop a message in the marketing channel so someone else can pick up on it? Thanks a bunch for this contribution

**Comment by cuchoi** [comment]
Thanks! Made the changes to the button [here](https://github.com/pymc-devs/pymcon/pull/36). Also added a favicon using the rocket from the PyMC logo.

**Pull request merged**